src/scenarios/shared/sod.py

```diff
@@ -67,7 +68,12 @@ def runtests(self, scenarioname, dirs, artifact=None):
         ]
         sod_args += dirs.split(';')
 
-        RunCommand(sod_args, verbose=True).run()
+        sod_command = RunCommand(sod_args, verbose=True)
+        sod_command.run()
+        
+        zero_size_regex = rf"({(scenarioname or 'Empty Scenario Name')}\s*-\s*Count\s*\|\s*0\.000\s*count)" # Checks if the overall count is zero    
+        if re.search(zero_size_regex, sod_command.stdout) != None:
```

Comment:
I'm not sure if searching stdout for a specific log is durable enough since it means that someone who changes the log ever in the sod executable may break this script without realising it. Instead, are we able to get this information by reading any of the files that are produced as output from the sod executable?

Reply:
Latest push switches from the regex/console output parsing to reading the json.